.github/workflows/lerna-publish.yml

```diff
@@ -54,7 +54,7 @@ jobs:
           docker run --name looker-sdk-codegen-ci -d -p 10000:9999 -p 20000:19999 us-west1-docker.pkg.dev/cloud-looker-sdk-codegen-cicd/looker/${{ fromJson(needs.setup.outputs.matrix_json)[0] }}
           docker logs -f looker-sdk-codegen-ci --until=30s &
 
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node@v4
         with:
           node-version: 18.x
```

Comment:
node 20?

Reply:
Nope. The action runs under node 20, but it sets up node 18.x here deliberately.